src/style.css

```diff
@@ -1 +1,95 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  --background-color: #e5e5e5;
+  --header-color: #fff;
+  --header-height: 60px;
+  --logo-height: 40px;
+  --logo-width: 40px;
+  --link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  background-color: var(--background-color);
+}
+
+.header {
```

Comment:
Please add padding for logo, instead of adding margin right/left for logo and nav
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/addc5b34-5541-4b19-b6a5-29170f077df0)